src/include/nmtkit/sampler.h

```diff
@@ -26,6 +26,11 @@ class Sampler {
   //   List of new samples.
   virtual std::vector<Sample> getSamples() = 0;
 
+  // Rerieves the size of the corpus.
+  // Returns:
+  //   Size of the corpus.
```

Comment:
This function might returns the number of "filtered" samples. It should be noted in the comment.